### PR TITLE
Update base image to cater vulnerabilities

### PR DESCRIPTION
**Pull request by nilasini**
## Purpose
Resolve https://github.com/wso2/docker-is/issues/299

### CODE REVIEW

File: dockerfiles/alpine/is/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Alpine Docker image
-FROM adoptopenjdk/openjdk11:jdk-11.0.10_9-alpine
+FROM adoptopenjdk/openjdk11:jdk-11.0.11_9-alpine
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
       com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.11.0.2"
```
Comment (RidmiR) [inline dockerfiles/alpine/is/Dockerfile:22]: Shall we update the latest release tag

File: dockerfiles/centos/is/Dockerfile [hunk 2]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK CentOS Docker image
-FROM adoptopenjdk/openjdk11:x86_64-centos-jdk-11.0.10_9
+FROM adoptopenjdk/openjdk11:x86_64-centos-jdk-11.0.11_9
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
       com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.11.0.2"
```
Comment (RidmiR) [inline dockerfiles/centos/is/Dockerfile:22]: Shall we update the latest release tag

File: dockerfiles/ubuntu/is/Dockerfile [hunk 3]
```diff
@@ -17,7 +17,7 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to AdoptOpenJDK Ubuntu Docker image
-FROM adoptopenjdk:11.0.10_9-jdk-hotspot-focal
+FROM adoptopenjdk:11.0.11_9-jdk-hotspot-focal
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
       com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v5.11.0.2"
```
Comment (RidmiR) [inline dockerfiles/ubuntu/is/Dockerfile:22]: Shall we update the latest release tag

File: dockerfiles/alpine/is/Dockerfile (hunk 1 above)
Comment (nilasini) [reply dockerfiles/alpine/is/Dockerfile:22]: jdk-11.0.11_9 is the latest release \[1]

\[1] https://hub.docker.com/r/adoptopenjdk/openjdk11/

Comment (RidmiR) [reply dockerfiles/alpine/is/Dockerfile:22]: No, This is our Docker IS release tag 
https://github.com/wso2/docker-is/releases/tag/v5.11.0.2
Its mentioned as 5.11.0.2. it should change into v5.11.0.5

**Review by RidmiR — APPROVED** [review]

**Pull request merged**